identifiers/country-ke/counties.csv

```diff
@@ -0,0 +1,48 @@
+id,name
+ocd-division/country:ke/county:01,Baringo

```

Comment:
I'd recommend using names instead of numbers in the identifier unless there's a good reason not to. So something like ocd-division/country:ke/county:baringo. Here and in counstituencies.csv. Otherwise looks good.

Reply:
I am using the ISO codes listed in here: https://www.iso.org/obp/ui/#iso:code:3166:KE 

So not sure if it would be ok to not apply them here (and just for context, I have been using ISO codes for other countries whenever they are available).

Reply:
Ah, sorry I missed that, for some reason I thought that was referencing the county name. Looks good to me then.